### PR TITLE
Update org references

### PR DESCRIPTION
**Pull request by jasnell**
/cc @mcollina @conoro

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -70,5 +70,5 @@ members of the project's leadership.
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
 available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
 
-[clinic]: mailto:clinic@nearform.com
+[clinic]: mailto:report@clinicjs.org
```
Comment (mcollina) [inline CODE_OF_CONDUCT.md:73]: does this email exist?

File: index.js [hunk 2]
```diff
@@ -178,7 +178,7 @@ class ClinicBubbleprof extends events.EventEmitter {
       headerLogo: logoFile,
       headerText: 'Bubbleprof',
       toolVersion: bubbleprofVersion,
-      nearFormLogo: nearFormLogoFile,
+      // nearFormLogo: nearFormLogoFile,
```
Comment (mcollina) [inline index.js:181]: just remove this line

File: index.js [hunk 3]
```diff
@@ -142,7 +142,7 @@ class ClinicBubbleprof extends events.EventEmitter {
     )
     // add logos
     const logoFile = fs.createReadStream(logoPath)
-    const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
+    // const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
```
Comment (mcollina) [inline index.js:145]: remove this line

**Review by mcollina — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (jasnell) [reply CODE_OF_CONDUCT.md:73]: Yes it does.

File: index.js [hunk 4]
```diff
@@ -124,7 +124,7 @@ class ClinicBubbleprof extends events.EventEmitter {
     const stylePath = path.join(__dirname, 'visualizer', 'style.css')
     const scriptPath = path.join(__dirname, 'visualizer', 'main.js')
     const logoPath = path.join(__dirname, 'visualizer', 'app-logo.svg')
-    const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
+    // const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
```
Comment (jasnell) [inline index.js:127]:
```suggestion
```

File: index.js (hunk 3 above)
Comment (jasnell) [inline index.js:145]:
```suggestion
```

File: index.js (hunk 2 above)
Comment (jasnell) [inline index.js:181]:
```suggestion
```

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**